rust/Cargo.toml.in

```diff
@@ -44,6 +44,7 @@ sha2 = "~0.9.2"
 digest = "~0.9.0"
 sha-1 = "~0.9.2"
```

Comment:
Should there be changes to doc ?
And a new files for event in the rules/ directory ?

Reply:
Will add new documentation of new app-proto format for eve.json to - doc/userguide/output/eve/eve-json-format.rst. Are there any other documentation elements I am forgetting about?

Would the new files for event in rules only be created if there are new detection keywords for this protocol?

Reply:
you can `git grep` some of the protocols to see the files where they are mentioned
something like `git grep -i ssh | cut -d: -f1 | uniq`

Reply:
> Would the new files for event in rules only be created if there are new detection keywords for this protocol?

the event in rules are not for detection keywords, but for protocol decoding errors or such

Reply:
Makes sense! How would I got about getting the page https://redmine.openinfosecfoundation.org/projects/suricata/wiki/AppLayer updated for the new SID changes introduced from this new event?

Reply:
I can do the wiki update when this will be ready